.eslintrc.cjs

```diff
@@ -39,5 +39,9 @@ module.exports = {
         'vue/one-component-per-file': 'off',
       },
     },
+    {
+      files: ['**/e2e/**/*.cy.ts', '**/e2e/cypress/**/*.ts'],
+      extends: 'plugin:cypress/recommended',
+    },
```

Comment:
See line 24-26. we already have that

Reply:
Good catch, that was a merge side effect.